File: SelectionSort.js
  selectionSort(arr,n){
   //code here
    for(let i =0; i<n;i++){
        for(let j = i+1; j< n; j++){
            if(arr[i]>arr[j]){
                let k = arr[i];
                arr[i] = arr[j];
                arr[j] = k;
            }
            

        }

    }

    return arr;
  }

File: Selection Sort.py
class Solution: 
    def select(self, arr, i):
        # code here 
        return None
        
    
    def selectionSort(self, arr,n):
        #code here
        for i in range(n):
            min_idx = i
            for j in range(i+1,n):
                if arr[min_idx] > arr[j]:
                    min_idx = j
            arr[i],arr[min_idx] = arr[min_idx],arr[i]
        return arr
